.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
```

Comment:
Thoughts on setting this up with a matrix to run tests across Node 14, 16, and 18?

Reply:
@erunion @dimitropoulos I just pushed this change, CI pipeline should now run for those major Node versions